src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -132,6 +133,18 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                         XContentParser parser = request.contentParser();
                         ensureExpectedToken(XContentParser.Token.START_OBJECT, parser.nextToken(), parser);
                         Map<String, Object> userDefaults = ParseUtils.parseStringToObjectMap(parser);
+
```

Comment:
might be my fault here since I pointed to wrong point in the code, but what if request has no content at all, we should still send this error message back right?

Reply:
Good point, I can move the validation logic around to cover both cases. There are some default use cases that have no required parameters